src/user/ZulipPasswordAuthView.js

```diff
@@ -20,7 +20,7 @@ import {
   attemptLogin,
 } from './userActions';
 
-export class ZulipPasswordAuthView extends React.Component {
+export class PasswordAuthView extends React.Component {
```

Comment:
There was an eslint issue where the exported default and the exported named component had the same name. This was correctly warned, because the first export is the pure component and the default one is the component wrapped with connect.

There are two improvements I see for later:
* No need to Zulip-ify the names (no Zulip in front of them)
* The component and the connect wrapped one can be in separate files, I have used this approach and it works really well. So in this case we can have PasswordAuthView and PasswordAuthContainer that imports PasswordAuthView. 'Container' shows that it takes care of retrieving data for the contained component, while the contained component is concerned with behavior and render only.